1-callbacks/solution-promisify.js

```diff
@@ -0,0 +1,29 @@
+async function solution() {
+  const users = ["Mary", "Daniel", "Stacy", "Gerardo", "Diego"];
+
+  const validateUser = require("./validate-user");
+  const { promisify } = require("util");
+
+  const successList = [],
+    failureList = [];
+  for (const user of users) {
+    const validateUserPromisify = promisify(validateUser);
+    try {
+      const result = await validateUserPromisify(user);
+      successList.push(result);
+    } catch (error) {
+      failureList.push(error.message);
+    }
+  }
+
+  console.log("Success\n");
+  for (const user of successList) {
+    console.log(`id: ${user.id} \nname: ${user.name}\n`);
+  }
+  console.log("Failure\n");
+  for (const message of failureList) {
+    console.log(message);
+  }
+}
```

Comment:
Nice job using Promisfy, async/await and try/cath 👍 😄 . Just same recommendations regarding, requieres, blank spaces and const declarations. :)

Reply:
thanks for make an extra effort for adding the solution with promisify  💪